### ISSUE TITLE
Homepage - Improve Focus support

### ISSUE THREAD

**Issue by perclasson** [material-components/material-components-flutter-gallery#569]
- On the homepage you should get focus on the settings icon first, then studies, then the demos.
- Settings should be settable with Focus (?).
